alphagov/paas-admin
# [#154558783] Allow user management

## What

We'd like to allow the org managers to pick up a user and edit their
permissions.

Some other changes have been made to satisfy the needs and approaches.
See commit messages for details.

## How to review

- Code review
- Run tests (should be done with travis)
- Run the application using rafal's environment
  ```
  OAUTH_AUTHORIZATION_URL="https://login.rafalp.dev.cloudpipeline.digital/oauth/authorize" OAUTH_TOKEN_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/token" OAUTH_CLIENT_ID="rafal-login" OAUTH_CLIENT_SECRET="secret" API_URL="https://api.rafalp.dev.cloudpipeline.digital" UAA_URL="https://uaa.rafalp.dev.cloudpipeline.digital" NOTIFY_API_KEY="not-important-for-that-story" npm start
  ```
- Attempt to list the users in single org and modify their permissions
